cegs_portal/search/views/v1/dna_features.py

```diff
@@ -53,9 +53,20 @@ def request_options(self, request):
                 * "name"
             feature_id
         """
+
+        def get_sig_only(value):
+            if value == "0" or value == "false" or value == "False":
+                return False
+            else:
+                return True
+
         options = super().request_options(request)
         options["assembly"] = request.GET.get("assembly", None)
         options["feature_properties"] = request.GET.getlist("property", [])
+        sig_only = request.GET.get("sig_only", True)
+        options["sig_only"] = get_sig_only(sig_only)
+        options["page"] = int(request.GET.get("page", 1))
+        options["per_page"] = int(request.GET.get("per_page", 2))
```

Comment:
The whole feature isn't paginated but some of the individual tables are. note that there is no "page" or "per_page" for the tables that already exist and are also paginated.